src/lib.rs

```diff
@@ -23,6 +23,17 @@ pub use cap::Cap;
 pub use ioctls::system::Kvm;
 pub use ioctls::vcpu::{VcpuExit, VcpuFd};
 pub use ioctls::vm::VmFd;
+
+// The following example is used to verify that our public
```

Comment:
The CpuId example can be moved to Cpuid::new(). I think it makes more sense in that context.

I have no ideas yet for `Result`. We might be able to test `KvmRunWrapper` once we add the high level documentation and give a more complex example of using this crate that is actually running some code and spawns the vCPU threads.

I would see this documentation test as a temporary solution and I would also add a TODO to fix it once we have all the documentation in place. What do you say?

Reply:
> I have no ideas yet for Result. 

Simply running an example that try to use the structure would test just that.

```Rust
/// # Example
///
/// ```
/// #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
/// use kvm_ioctls::{KvmRunWrapper, Result};
/// ```

```

Reply:
@andreeaflorescu I moved the `CpuId` example to the ioctl module, and kept the `use kvm_ioctls::{KvmRunWrapper, Result};` example. This does test the export of those structures.

Reply:
My concern was only that the lines testing the imports are not placed somewhere where it makes sense from the crate documentation point of view. What I was suggesting was to remove this example once we have other code examples that use `KvmRunWrapper` and `Result` for something meaningful.

Reply:
Issue #17 created to track that.
The PR has also been updated to move the `CpuId` example next to the `new` method.